terragrunt/env/staging/api/terragrunt.hcl

```diff
@@ -3,7 +3,7 @@ terraform {
 }
 
 dependencies {
-  paths = ["../network", "../ecr"]
+  paths = ["../network", "../ecr", "../dynamodb"]
```

Comment:
Perfect!  Could you also add this same config to the `env/production/api/terragrunt.hcl` when you get a chance

Reply:
Sorry! Clearly need more coffee this morning. It has been added.